WorkerForm.cs

```diff
@@ -869,7 +870,11 @@ public void UploadGames()
                         case MainForm.ConsoleType.SNES:
                         case MainForm.ConsoleType.SuperFamicom:
                             originalSyncCode = $"mkdir -p \"{rootFsPath}{gamesPath}/{originalGames[originalCode]}/{originalCode}/\" && " +
-                                $"rsync -ac \"{gamesPath}/{originalCode}/\" \"{rootFsPath}{gamesPath}/{originalGames[originalCode]}/{originalCode}/\" && " +
+                                $"ln -s \"{squashFsPath}{gamesPath}/{originalCode}/{originalCode}.png\" \"{rootFsPath}{gamesPath}/{originalGames[originalCode]}/{originalCode}/{originalCode}.png\" &&" +
+                                $"ln -s \"{squashFsPath}{gamesPath}/{originalCode}/{originalCode}_small.png\" \"{rootFsPath}{gamesPath}/{originalGames[originalCode]}/{originalCode}/{originalCode}_small.png\" &&" +
+                                $"ln -s \"{squashFsPath}{gamesPath}/{originalCode}/{originalCode}.sfrom\" \"{rootFsPath}{gamesPath}/{originalGames[originalCode]}/{originalCode}/{originalCode}.sfrom\" &&" +
+                                $"ln -s \"{squashFsPath}{gamesPath}/{originalCode}/autoplay/\" \"{rootFsPath}{gamesPath}/{originalGames[originalCode]}/{originalCode}/autoplay\" &&" +
+                                $"cp \"{gamesPath}/{originalCode}/{originalCode}.desktop\" \"{rootFsPath}{gamesPath}/{originalGames[originalCode]}/{originalCode}/{originalCode}.desktop\" && " +

```

Comment:
you only need cp desktop actually, ln -s are unneeded. but check if it works, because I didn't. sed /usr/share/games/ -> /var/lib/hakchi/squashfs/usr/share/games

Reply:
Great idea but need to make sure that /var/lib/hakchi/squashfs is mounted.

Reply:
@ClusterM Check to see if it is and if not, re-install the mod scripts to the NAND?

Also, this same modification should work on the NES Classic with some changes for the different files, but I don't have one to test.